data/basics/userBalance.go

```diff
@@ -146,8 +146,9 @@ type AccountData struct {
 	// the past week".
 	RewardedMicroAlgos MicroAlgos `codec:"ern"`
 
-	VoteID      crypto.OneTimeSignatureVerifier `codec:"vote"`
-	SelectionID crypto.VRFVerifier              `codec:"sel"`
+	VoteID       crypto.OneTimeSignatureVerifier `codec:"vote"`
+	SelectionID  crypto.VRFVerifier              `codec:"sel"`
+	BlockProofID crypto.VerifyingKey             `codec:"blockproof"`
```

Comment:
consider using a shorter codec field name. This field name is included in every single msgp encoding of the account data, which is quite frequent. maybe `blkprf` ?

Reply:
Thanks for the comments! 
i've changed it